auctions/models.py

```diff
@@ -339,7 +333,8 @@ class Payment(models.Model):
         ('Stripe', 'Stripe'),
         ('PayPal', 'PayPal'),
     )
-    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False),
+    id = models.UUIDField(
+        primary_key=True, default=uuid.uuid4(), editable=False),
```

Comment:
Should be able to use `default=uuid.uuid4`, or something like `default=generate_id` and specify a `def generate_id` function.